azuread/resource_group_member.go

```diff
@@ -0,0 +1,113 @@
+package azuread
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/graph"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/ar"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/validate"
+)
+
+func resourceGroupMember() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGroupMemberCreate,
+		Read:   resourceGroupMemberRead,
+		Delete: resourceGroupMemberDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"group_object_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.UUID,
+			},
+			"member_object_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.UUID,
+			},
+		},
+	}
+}
+
+func resourceGroupMemberCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).groupsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	groupID := d.Get("group_object_id").(string)
+	memberID := d.Get("member_object_id").(string)
+
+	if err := graph.GroupAddMember(groupID, memberID, client, ctx); err != nil {
+		return err
+	}
+
+	id := fmt.Sprintf("%s/%s", groupID, memberID)
```

Comment:
Could we change this to:
```suggestion
	id := fmt.Sprintf("%s/member/%s", groupID, memberID)
```